tests/Integration/Migration/ResetDatabaseWithMigrationTest.php

```diff
@@ -45,16 +45,9 @@ public function it_generates_valid_schema(): void
         $application = new Application(self::bootKernel());
         $application->setAutoExit(false);
 
-        $exit = $application->run(
-            new ArrayInput(['command' => 'doctrine:schema:validate', '-v' => true]),
-            $output = new BufferedOutput()
-        );
-
-        // The command actually fails, because of a bug in doctrine ORM 3!
-        // https://github.com/doctrine/migrations/issues/1406
```

Comment:
the bug is fixed :tada: 
https://github.com/doctrine/migrations/issues/1406#issuecomment-2596839269